src/DataWrangling.jl/interpolate_bathymetry.jl

```diff
@@ -24,27 +24,40 @@ SpectralInterpolation(; filter_func = (l) -> exp(-l * (l+1)/ 180 / 240), spectra
 
 
 """
-    interpolate_bathymetry_from_file(resolution, latitude; 
+    interpolate_bathymetry_from_file(resolution, maximum_latitude; 
```

Comment:
Should we use `regrid!` to do this conservatively?

Reply:
we can, but it might not be so important for bathymetry as for IC and BC

Reply:
I think we do typically want to set the bathymetric height to be the "average" height in some region for some dynamics (large scale Rossby waves, tides?) But perhaps even more it might be nice to use fewer functions though to keep the code easier to understand (if the two approaches are equivalent)